docs/proposals/service-discovery/README.md

```diff
@@ -47,31 +50,34 @@ Following are flows to support the service import proposal:
 1. `Deployment` and `Service` are created on cluster member1 and the `Service` imported to cluster member2 using `ServiceImport` (described below as [user story 1](#story-1))
 2. `Deployment` and `Service` are created on cluster member1 and both propagated to cluster member2. `Service` from cluster member1 is imported to cluster member2 using `ServiceImport` (described below as [user story 2](#story-2))
 
-The proposal for this flow is what can be referred to as local-and-remote service discovery. The options could be:
+The proposal for this flow is what can be referred to as local-and-remote service discovery. In the process handling, it can be simply distinguished into the following scenarios:
 
 1. **Local** only - In case there's a local service by the name `foo` Karmada never attempts to import the remote service and doesn't create an `EndPointSlice`
 2. **Local** and **Remote** - Users accessing the `foo` service will reach either member1 or member2
 3. **Remote** only - in case there's a local service by the name `foo` Karmada will remove the local `EndPointSlice` and will create an `EndPointSlice` pointing to the other cluster (e.g. instead of resolving member2 cluster is will reach member1)
 
-This proposal suggests to distinguish the services type by the extending the API `multicluster.x-k8s.io/v1alpha1` of `ServiceImport` with Karmada specific annotations:
+Based on the above three scenarios, we have proposed two strategies:
+
+- **RemoteAndLocal** - When accessing Service, the traffic will be evenly distributed between the local cluster and remote cluster's Service.
+- **LocalFirst** - When accessing Services, if the local cluster Service can provide services, it will directly access the Service of the local cluster. If a failure occurs in the Service on the local cluster, it will access the Service on remote clusters.
+
+This proposal suggests using the [MultiClusterService API](https://github.com/karmada-io/karmada/blob/24bb5829500658dd1caeea16eeace8252bcef682/pkg/apis/networking/v1alpha1/service_types.go#L30) to enable cross-cluster service discovery. To avoid conflicts with the previously provided [prefixed cross-cluster service discovery](./../networking/multiclusterservice.md#cross-clusters-service-discovery), we can add an annotation on the MultiClusterService API with the key `discovery.karmada.io/strategy`,  its value can be either `RemoteAndLocal` or `LocalFirst`.
+
 ```yaml
-apiVersion: multicluster.x-k8s.io/v1alpha1
-kind: ServiceImport
+apiVersion: networking.karmada.io/v1alpha1
+kind: MultiClusterService
 metadata:
-  name: serve
-  namespace: demo
-  annotations:
-    discovery.karmada.io/strategy: LocalAndRemote
+   name: foo
+   annotation:
+      discovery.karmada.io/strategy: RemoteAndLocal 
 spec:
-  type: ClusterSetIP
-  ports:
-  - port: 80
-    protocol: TCP
+   types:
+      - CrossCluster
```

Comment:
The CrossCluster type seems to be ambiguous.

Reply:
Do you have any suggestions?